src/main/scala/streaming/SampleStatusesHashtagPrinter.scala

```diff
@@ -6,6 +6,7 @@ import com.danielasfregola.twitter4s.entities.Tweet
 
 object SampleStatusesHashtagPrinter extends App {
 
+  // TODO - Make sure to define your consumer and access tokens and have firehose permissions!
   val client = TwitterStreamingClient()
```

Comment:
[Unrelated to this PR]. L12 + L20 we should just map instead of having weird stuff like `case None => None` or `case None => ()`.

This has been fixed ;)